go.mod

```diff
@@ -1,67 +1,68 @@
 module github.com/operator-framework/operator-sdk
 
-go 1.17
+go 1.18
 
 require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/fatih/structtag v1.1.0
-	github.com/go-logr/logr v1.2.0
+	github.com/go-logr/logr v1.2.2
 	github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0
 	github.com/iancoleman/strcase v0.2.0
 	github.com/kr/text v0.2.0
 	github.com/markbates/inflect v1.0.4
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.2.2
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.17.0
-	github.com/operator-framework/api v0.14.1-0.20220413143725-33310d6154f3
-	github.com/operator-framework/helm-operator-plugins v0.0.11-0.20220607145946-5b29693d1612
-	github.com/operator-framework/java-operator-plugins v0.1.0
-	github.com/operator-framework/operator-lib v0.10.0
-	github.com/operator-framework/operator-manifest-tools v0.2.0
-	github.com/operator-framework/operator-registry v1.21.0
+	github.com/onsi/gomega v1.18.1
```

Comment:
Does this one need a changelog?

Reply:
This shouldn't need a changelog since this is just internal for OSDK tests